documentation/src/main/asciidoc/migration/index.adoc

```diff
@@ -65,11 +65,197 @@ older indexes can be read from and written to without reindexing.
 [[outboxpolling]]
 === Outbox polling database tables
 
-The event and agent database tables used for
-link:https://docs.jboss.org/hibernate/search/{hibernateSearchVersionShort}/reference/en-US/html_single/#coordination-database-polling[outbox-polling]
-in Hibernate Search {hibernateSearchVersion}
-are backward-compatible with Hibernate Search {hibernateSearchPreviousStableVersionShort}:
-no database schema update is necessary for these tables.
+If you use the incubating link:{hibernateSearchDocUrl}#coordination-outbox-polling[`outbox-polling` coordination strategy],
+you may be impacted by the changes to entity mapping for entities that represents the outbox event and agent,
+requiring database schema changes.
+
+Changes are only applicable if the `hibernate.search.coordination.entity.mapping.*.uuid_type` was set to `default` (implicitly or explicitly).
+In case when schema changes are not possible to apply at the moment, the configuration settings can be provided
+to specify the type to represent UUIDs in the database:
+
+[source, properties]
+----
+hibernate.search.coordination.entity.mapping.outboxevent.uuid_type=CHAR
+hibernate.search.coordination.entity.mapping.agent.uuid_type=CHAR
+----
+
+The `id` column changes its type from the current one to the one used by the corresponding Hibernate ORM Database dialect
+by default for representing the uuid type, except the following databases where no changes are expected:
```

Comment:
I must admit I have a hard time understanding this sentence... Maybe rephrase this way, and add a link to the ORM docs?

```suggestion
The default type for UUIDs in outbox-polling tables now match
https://docs.jboss.org/hibernate/orm/6.4/userguide/html_single/Hibernate_User_Guide.html#basic-uuid[the default type for UUIDs in Hibernate ORM].

The default behavior should not change for the following databases:
```